src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Properties/launchSettings.json

```diff
@@ -23,7 +23,7 @@
       "launchUrl": "authentication/swagger",
       "applicationUrl": "http://localhost:5040",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development",
+        "ASPNETCORE_ENVIRONMENT": "Production",
```

Comment:
This would only affect us when running the component locally.. Could we consider using staging instead so that it matches AT

Reply:
I changed it back to Development. I was using Production to test the server side redirect to the error endpoint.